src/Nethermind/Nethermind.Core/Caching/MemCountingCache.cs

```diff
@@ -155,10 +155,10 @@ public void Delete(Keccak key)
         private void Replace(Keccak key, byte[] value)
         {
             LinkedListNode<LruCacheItem>? node = _lruList.First;
-            
+
             // ReSharper disable once PossibleNullReferenceException
-            MemorySize += MemorySizes.Align(value.Length) - MemorySizes.Align(node?.Value.Value.Length ?? 0);
-            
+            MemorySize += MemorySizes.Align(value.Length) - MemorySizes.Align(node!.Value.Value.Length);
+
```

Comment:
I realized this had a comment disabling the check above it, so I think `!` is a more correct solution.